lib/openhab/core/items/semantics.rb

```diff
@@ -662,35 +666,62 @@ def equipments(*types, subclasses: true)
   # @example Get all the power switch items for every equipment in a room
   #   lGreatRoom.equipments.members.points(Semantics::Switch)
   #
+  # @example Get (all) the temperature setpoints for a thermostat
+  #   eMainFloorThermostat.points(Semantics::Setpoint, Semantics::Temperature)
+  #
+  # @example Get all the brightness and color (light) points in a room
+  #   lKitchen.equipments.members.points(Semantics::Brightness, Semantics::Color)
+  #
+  # @example Get all the brightness and color (light) control points in a room
+  #   lKitchen.equipments.members.points(Semantics::Control, Semantics::Brightness, Semantics::Color)
+  #
+  # @example Get the air temperature and speed control points in a room
+  #   lGreatRoom.equipments.members.points([Semantics::Measurement, Semantics::Temperature],
+  #                                        [Semantics::Control, Semantics::Speed])
+  #   # => [GreatRoom_AirTemp, GreatRoom_FanSpeed]
+  #
   # @see #members
   #
   def points(*point_or_property_types, subclasses: true)
     unless (0..2).cover?(point_or_property_types.length)
```

Comment:
Why is it limited to 2? The example shows a use of 3 (but no specs for this case)

Can we support:
`points(Semantics::Control, Semantics::Brightness, Semantics::Color, Semantics::Power, Semantics::ColorTemperature)`

Or are we meant to use the array syntax, in which case:
`points([Semantics::Control, Semantics::Brightness], [Semantics::Control, Semantics::Color], [Semantics::Control, Semantics::Power], [Semantics::Control, Semantics::ColorTemperature]) `

Reply:
Good catch. It should no longer be limited to 2. Will fix.